### PR TITLE
make IsPojo support methods defined in superclasses

### PR DESCRIPTION
**Pull request by pettermahlen**
I've needed this - but maybe there is a reason to not permit it?

### CODE REVIEW

File: pojo/src/main/java/com/spotify/hamcrest/pojo/IsPojo.java [hunk 1]
```diff
@@ -64,7 +64,7 @@
     return where(
         methodName,
         self -> {
-          final Method method = self.getClass().getDeclaredMethod(methodName);
```
Comment (dflemstr) [inline pojo/src/main/java/com/spotify/hamcrest/pojo/IsPojo.java:67]: This technically makes us no longer support private/package private methods; add test for that?

**Review by dflemstr — CHANGES_REQUESTED** [review]

File: pojo/src/main/java/com/spotify/hamcrest/pojo/IsPojo.java (hunk 1 above)
Comment (pettermahlen) [reply pojo/src/main/java/com/spotify/hamcrest/pojo/IsPojo.java:67]: Fixed

**Review by dflemstr — APPROVED** [review]

**Comment by dflemstr** [comment]
:+1: I like @pettermahlen

**Comment by pettermahlen** [comment]
Great - I don't have merge access, so please merge at will. :)

**Pull request merged**